docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -366,6 +366,46 @@ For example, select the top 10 percent of nations by population::
         )
       );
 
+.. _sqlserver-procedure-function:
+
+``procedure(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``procedure`` function allows you to run stored procedure on the underlying
+database directly. It requires syntax native to SQL Server.
+
+.. note::
+
+    StoredProcedures which return multiple statement, with a non-select statement, with
+    output parameters, or with an If-Else branch are not supported.
+
+
+For example, run the stored procedure ``employee_sp`` in the ``example`` catalog and the
```

Comment:
"For example, run the stored procedure" -> "The follow example runs the stored procedure"